src/openrct2/drawing/ImageId.hpp

```diff
@@ -43,22 +43,22 @@ struct ImageId
 {
 private:
     // clang-format off
-    static constexpr uint32_t MASK_INDEX       = 0b00000000000001111111111111111111;
-    static constexpr uint32_t MASK_REMAP       = 0b00000111111110000000000000000000;
-    static constexpr uint32_t MASK_PRIMARY     = 0b00000000111110000000000000000000;
-    static constexpr uint32_t MASK_SECONDARY   = 0b00011111000000000000000000000000;
-    static constexpr uint32_t FLAG_PRIMARY     = 0b00100000000000000000000000000000;
-    static constexpr uint32_t FLAG_BLEND       = 0b01000000000000000000000000000000;
-    static constexpr uint32_t FLAG_SECONDARY   = 0b10000000000000000000000000000000;
-    static constexpr uint32_t SHIFT_REMAP      = 19;
-    static constexpr uint32_t SHIFT_PRIMARY    = 19;
-    static constexpr uint32_t SHIFT_SECONDARY  = 24;
-    static constexpr uint32_t INDEX_UNDEFINED  = 0b00000000000001111111111111111111;
-    static constexpr uint32_t VALUE_UNDEFINED  = INDEX_UNDEFINED;
-
-    static constexpr uint8_t  NEW_FLAG_PRIMARY      = 1;
-    static constexpr uint8_t  NEW_FLAG_BLEND        = 2;
-    static constexpr uint8_t  NEW_FLAG_SECONDARY    = 4;
+    static constexpr uint32_t kMaskIndex       = 0b00000000000001111111111111111111;
+    static constexpr uint32_t kMaskRemap       = 0b00000111111110000000000000000000;
+    static constexpr uint32_t kMaskPrimary     = 0b00000000111110000000000000000000;
+    static constexpr uint32_t kMaskSecondary   = 0b00011111000000000000000000000000;
+    static constexpr uint32_t kFlagPrimary     = 0b00100000000000000000000000000000;
+    static constexpr uint32_t kFlagBlend       = 0b01000000000000000000000000000000;
+    static constexpr uint32_t kFlagSecondary   = 0b10000000000000000000000000000000;
+    static constexpr uint32_t kShiftRemap      = 19;
+    static constexpr uint32_t kShiftPrimary    = 19;
+    static constexpr uint32_t kShiftSecondary  = 24;
```

Comment:
I think these are all obsolete, right? Perhaps we should remove them instead.

Reply:
Removed them.